docs/deployment/openshift/README.md

```diff
@@ -1,83 +1,84 @@
 # Deployment Guide for OpenShift
-This document will guide you through the installation of falcon-operator and deployment of either the:
-- [FalconContainer](resources/container/README.md) custom resource to the cluster with Falcon Container image being mirrored from CrowdStrike container registry to OpenShift ImageStreams (on cluster registry).
-- [FalconNodeSensor](resources/node/README.md) custom resource to the cluster.
 
```

Comment:
Please keep these in the docs... especially since the operator links here for openshift-specific docs and no there is no easy way to find the CR docs.